.gitignore

```diff
@@ -25,3 +25,5 @@ __snapshots__
 
 # debug & other vs code settings
 .vscode
+server/uploads/
```

Comment:
Why have these files been added to the .gitignore? :)

Reply:
Oh yes, @rajivtitus  I was working with testing the savePhoto controller and all the images were getting saved it ended up being 23mb and lots of files, So should I delete all images in upload folder and remove it from gitignore  ??

Reply:
If it a folder only to store the images, I guess that's fine :)